src/sweetalert2.scss

```diff
@@ -160,6 +160,14 @@ body {
     align-items: flex-end;
     justify-content: flex-end;
   }
+  
```

Comment:
trailing spaces here

Reply:
tip: if you're using VSCode, the setting is `"files.trimTrailingWhitespace": true`

Reply:
I will change the request in a few hours. I‘m currently not at home.